src/helpers/enhancedBrowserMetrics.js

```diff
@@ -98,6 +98,26 @@ function getDeviceDimensions(window) {
   return {height, width};
 }
 
+/* eslint-disable */
```

Comment:
Can we add a comment as to why we want to disable eslint for this block here? Alternatively, if it's for the `window` variable, I think we could add an `/* eslint-env browser */` to the top of the file?

Reply:
removed it